lib/testmap.py

```diff
@@ -153,14 +140,15 @@
     },
     'candlepin/subscription-manager': {
         'main': [
-            'rhel-8-4',
+            'rhel-9-0',
         ],
-        'subscription-manager-1.28': [
```

Comment:
this branch is "8 devel", so please leave it, and add 8.5 to it (leaving 8.4 for checking of potential 8.4 backports)